randovania/games/super_metroid/gui/preset_settings/super_patches_tab.py

```diff
@@ -0,0 +1,161 @@
+import dataclasses
+import functools
+
+from PySide2 import QtWidgets
+from PySide2.QtWidgets import QComboBox
+
+from randovania.gui.generated.preset_patcher_super_patches_ui import Ui_PresetPatcherSuperPatches
+from randovania.gui.preset_settings.preset_tab import PresetTab
+from randovania.interface_common.preset_editor import PresetEditor
+from randovania.layout.preset import Preset
+from randovania.games.super_metroid.layout.super_metroid_patch_configuration import MusicMode
+
+class PresetSuperPatchConfiguration(PresetTab, Ui_PresetPatcherSuperPatches):
+    def __init__(self, editor: PresetEditor):
+        super().__init__(editor)
+        self.setupUi(self)
+        self.colorblind_checkbox.stateChanged.connect(functools.partial(
```

Comment:
What I'd do here is something like this:
```python
checkboxes = {
    "colorblind_mode": self.colorblind_checkbox,
    "instant_g4": self.instant_g4_checkbox,
    ...
}
for name, checkbox in checkboxes.items():
    checkbox.stateChanged.connect(functools.partial(self._on_patch_checkbox_changed, name))
```

Reply:
I... don't know why this never occurred to me. Thanks for the suggestion, it seems so obvious in hindsight.